build-in-container/scripts/setup.sh

```diff
@@ -59,6 +59,37 @@ check_specs() {
     fi
 }
 
+# enable custom-repo.repo to install RPMs from
+setup_custom_repofile() {
+    echo "------------ Setting up custom repofile ------------"
+    echo -e "\n" >> $MARINER_BASE_DIR/toolkit/resources/manifests/package/local.repo
```

Comment:
The `local.repo` file is expected to be completely static. What are the shortcomings with the toolkit that necessitate this approach?

Reply:
From what I understand, pkgworker gets the path to local.repo here https://github.com/microsoft/CBL-Mariner/blob/main/toolkit/tools/pkgworker/pkgworker.go#L42 and copies it into the chroot https://github.com/microsoft/CBL-Mariner/blob/main/toolkit/tools/pkgworker/pkgworker.go#L175. I too would not prefer to append the contents of custom-repo.repo to local.repo, but I am not sure how to get custom-repo.repo into chroot

Reply:
The tools should be passing in everything listed in `REPO_LIST="/path/to/first.repo /path/to/second.repo` to the workers.